docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,138 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
```

Comment:
Change this alias and title to use an imperative: `configure-helm-ha-deduplication-consul`.